### PR TITLE
feat: Results get endpoint

### PR DESCRIPTION
**Pull request by Ansaar-Syed**
## Pre-Requisites
- [x] Yes, I updated [Authors.md](../blob/main/Authors.md) **OR** this is not my first contribution
- [x] Yes, I included and/or modified tests to cover relevent code **OR** my change is non-technical
- [x] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../blob/main/THIRD-PARTY-NOTICES.txt)

### CODE REVIEW

File: packages/api/src/api/expoJudgingSession/index.ts [hunk 1]
```diff
@@ -10,3 +10,4 @@ export const expoJudgingSession = Router();
 expoJudgingSession.post('', adminMiddleware, post);
 expoJudgingSession.get('', adminMiddleware, get);
 expoJudgingSession.use('/:id', judgeMiddleware, id);
+expoJudgingSession.use('/:id', adminMiddleware, id);
```
Comment (charliealbright) [inline packages/api/src/api/expoJudgingSession/index.ts:13]: What's the need for this?

Comment (SpencerKaiser) [reply packages/api/src/api/expoJudgingSession/index.ts:13]: The only folks that have access to results should be admins so this adds admin middleware to the route... however, this is not the right way to do it...

@Ansaar-Syed you'll need to remove `judgeMiddleware` from the line above this and have the route itself manage it's middleware one level higher up the hierarchy

Comment (Ansaar-Syed) [reply packages/api/src/api/expoJudgingSession/index.ts:13]: done

File: packages/api/src/api/expoJudgingSession/id/results/index.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { Router } from 'express';
+import { get } from './get';
+
+export const results = Router();
```
Comment (SpencerKaiser) [inline packages/api/src/api/expoJudgingSession/id/results/index.ts:4]: Here's how I know you didn't test this method 😉 
```suggestion
export const results = Router({ mergeParams: true });
```

File: packages/api/tests/api/expoJudgingSession/index.test.ts [hunk 3]
```diff
@@ -54,15 +52,15 @@ describe('/expoJudgingSession post endpoint registration', () => {
     });
   });
 
-  it('uses adminMiddleware and registers the GET route for the handler', async () => {
-    await jest.isolateModulesAsync(async () => {
-      const { expoJudgingSession } = await import('../../../src/api/expoJudgingSession');
+  // it('uses adminMiddleware and registers the GET route for the handler', async () => {
```
Comment (SpencerKaiser) [inline packages/api/tests/api/expoJudgingSession/index.test.ts:55]: Delete this if it's no longer being used

**Review by SpencerKaiser — CHANGES_REQUESTED** [review]
Super close, just a few changes

**Review by SpencerKaiser — APPROVED** [review]

**Pull request merged**